lib/kafka_ex.ex

```diff
@@ -29,10 +31,10 @@ defmodule KafkaEx do
   def create_worker(name, worker_init \\ [])
 
   def create_worker(name, worker_init) do
-    worker_init = case worker_init do
-      [] -> [uris: Application.get_env(:kafka_ex, :brokers)]
-      _   -> worker_init
-    end
+    defaults = [uris: Application.get_env(:kafka_ex, :brokers),
+                consumer_group: Application.get_env(:kafka_ex, :consumer_group)]
+
+    worker_init = Keyword.merge(defaults, worker_init)
```

Comment:
I understand what `Keyword.merge/2` does, first pass through the code made it look like a stylistic change (since there are a couple of them in this PR), but we still have to support users not using `consumer_groups` hence the code without the default `consumer_group` is not different from the new implementation